### DISCUSSION TITLE
future plans on statistics

### DISCUSSION THREAD

**Discussion by volleyfreak** [shukkkur/VolleyVision#10] [Q&A]
Hey Shukkur,
First of all: Great work!
I am currently working on a thesis about object detection and automated statistics in volleyball and your volleyball dataset was a great help. ❤️

Do you know about https://www.balltime.com/?
I just stumbled across it, and they seem to have pretty good automated statistics. Sadly it is not open source.
You mentioned that your ultimate goal is to create statistics as well. Any updates on this?

**Comment by shukkkur** [comment]
Hi @volleyfreak ! Thanks for your kind words. Is the thesis for bachelor or master degree? 

Yep, I've seen `Balltime` and indeed they are doing a great job. And yes, it obviously wouldn't be open-source, after all everyone aims at commercializing their products )))   

As for statistics, I was planning but it isn't as easy as I thought of it. For now, I haven't made any progress. But, I recently have realized the power of research papers) For example, now I am looking into [this paper](http://www.cs.toronto.edu/~namdar/pdfs/sports_cvpr_2017.pdf) so I can imporve court detection & tracking. 

So, unfortunately the only thing I can tell you is to try look for papers and second option would be trying to combine different parts of VolleyVision. For example, theoretically, combining court detection with ball tracking you _could_ create **Smart Ball In/Out System**. haha, easier said that done. 

Anyway, good luck and look forward to any discoveries on your side.

**Comment by volleyfreak** [comment]
My thesis will be for master degree. I had some success on tracking players already and I hope I can create some simple statistics at the end. 
Thanks!
If there is anything I can publish at the end, I will provide you with an update and link to my repo. :+1:

**Reply by volleyfreak** [reply]
I finally finished my thesis. 
If you are interested, you can check out my [blog post](https://www.inovex.de/de/blog/data-driven-volleyball-creating-player-statistics-using-computer-vision/) on my company's site. In the end, I was able to generate some simple statistics, also by using parts of your datasets. So thanks again here for your work and inspiration.
And I have plans to further develop and publish my system if I find time.. (not soon though)
